mm2src/coins/eth.rs

```diff
@@ -5139,7 +5157,12 @@ pub async fn eth_coin_from_conf_and_request(
     }
     let contract_supports_watchers = req["contract_supports_watchers"].as_bool().unwrap_or_default();
 
-    let (my_address, key_pair) = try_s!(build_address_and_priv_key_policy(conf, priv_key_policy).await);
+    let path_to_address = try_s!(json::from_value::<Option<StandardHDCoinAddress>>(
+        req["path_to_address"].clone()
+    ))
+    .unwrap_or_default();
```

Comment:
debug!(r#"Failed to get "path_to_address" from request, default one is in use"#);

Reply:
Debug logs are not available for any other request fields deserializations and for other coins/methods that use a struct for requests instead of `Json`. I would like to keep the behaviour consistent throughout the code, but this also will be changed as this PR is intermediate.